Project 1/styles.css

```diff
@@ -0,0 +1,20 @@
+.card{
```

Comment:
Same note about Beautify. This plugin should help format your files and is required.

Reply:
You may want to consider adding some padding to your `.card` element so the text is not right up on its edge:
![padding](https://user-images.githubusercontent.com/2653980/54493518-699ac600-48a7-11e9-9192-62595a93db0e.png)